docs/libs/creating-lib.md

```diff
@@ -105,6 +106,28 @@ module.exports = function (config) {
 };
 ```
 
+- `project.json`
+
+  - add the lint `targets > lint`
+
+    replace the TODO: with the library name
+
+    ```json
+    "lint": {
+      "executor": "@angular-eslint/builder:lint",
+      "options": {
+        "lintFilePatterns": [
+          "integration-libs/TODO:/**/*.ts",
+          "integration-libs/TODO:/**/*.html"
+        ]
+      }
+    }
+    ```
+
+  - add the tag(s)
+    - type:feature
+    - type:integration `IF AND ONLY IF` it is an integration lib.
```

Comment:
An example of tags in the json above would be helpful

Reply:
done